bin/metrics-mesos.rb

```diff
@@ -33,6 +33,10 @@
 require 'socket'
 require 'json'
 
+# Mesos default ports are defined here: http://mesos.apache.org/documentation/latest/configuration
+MASTER_DEFAULT_PORT = '5050'
+SLAVE_DEFAULT_PORT = '5051'
```

Comment:
@analytically @lusid  There are two schools of thoughts here. We can leave them defined as the options make no distinction between slave/master so we define them outside.

We could create a new options one for each case but that could be ugly.  As the code is now in the commit is fine for the use case.